src/tests/ftest/util/soak_utils.py

```diff
@@ -459,6 +460,119 @@ def launch_vmd_identify_check(self, name, results, args):
+        ranks = ",".join(str(rank) for rank in ranklist)
+        # init the status dictionary
+        params = {"name": name,
+                  "status": status,
+                  "vars": {"host": reboot_host, "ranks": ranklist}}
+        self.log.info(
+            "<<<PASS %s: %s started on ranks %s at %s >>>\n", self.loop, name, ranks, time.ctime())
+        # reboot host in 1 min
+        result = run_remote(self.log, reboot_host, "sudo shutdown -r +1")
+        if result.passed:
+            status = True
+        else:
+            self.log.error(f"<<<FAILED:{name} - {reboot_host} failed to issue reboot")
+            status = False
+
+        if not wait_for_result(self.log, check_ping, 90, 5, True, host=reboot_host,
+                               expected_ping=False, cmd_timeout=60, verbose=True):
+            self.log.error(f"<<<FAILED:{name} - {reboot_host} failed to reboot")
+            status = False
+
+        if status:
+            rebuild_status = True
+            for pool in pools:
+                rebuild_status &= wait_for_pool_rebuild(self, pool, name)
+            status = rebuild_status
+
+    elif name == "REBOOT_REINTEGRATE" and self.harasser_results["REBOOT"]:
+        reboot_host = self.harasser_args["REBOOT"]["host"]
+        ranklist = self.harasser_args["REBOOT"]["ranks"]
+        ranks = ",".join(str(rank) for rank in ranklist)
+        self.log.info("<<<PASS %s: %s started on host %s at %s>>>\n", self.loop, name, reboot_host,
+                      time.ctime())
+
+        self.dmg_command.system_query()
+        # wait for node to complete rebooting
+        if not wait_for_result(self.log, check_ping, 60, 5, True, host=reboot_host,
+                               expected_ping=True, cmd_timeout=60, verbose=True):
+            self.log.error(f"<<<FAILED:{name} - {reboot_host} failed to reboot")
+            status = False
+        if not wait_for_result(self.log, check_ssh, 120, 2, True, hosts=reboot_host,
+                               cmd_timeout=30, verbose=True):
+            self.log.error(f"<<<FAILED:{name} - {reboot_host} failed to reboot")
+            status = False
+        # issue a restart
+        self.log.info("<<<PASS %s: Issue systemctl restart daos_server on %s at %s>>>\n",
+                      self.loop, name, reboot_host, time.ctime())
+        cmd_results = run_remote(self.log, reboot_host, "sudo systemctl restart daos_server")
+        if cmd_results.passed:
+            time.sleep(30)
+            try:
+                self.dmg_command.system_start(ranks=ranks)
+                status = True
+            except TestFail as error:
+                self.log.error("<<<FAILED:dmg system start failed", exc_info=error)
+                status = False
+            for pool in pools:
+                self.dmg_command.pool_query(pool.identifier)
+        if status:
+            # Wait ~ 30 sec before issuing the reintegrate
+            time.sleep(30)
```

Comment:
It's a good idea to print a message that we're sleeping. Otherwise, the job.log might look like something is hanging

Reply:
Do we actually need this?